reference/library/src/morpheus-master/sass/base/_defaults.scss

```diff
@@ -107,3 +107,29 @@ u, ins {
 s, strike, del {
 	text-decoration: line-through;
 }
+
+input[type="submit"][disabled="disabled"],
+input[type="button"][disabled="disabled"],
+input[type="submit"][disabled="true"],
+input[type="button"][disabled="true"],
+input[type="submit"][disabled],
+input[type="button"][disabled],
+input[type="submit"][class="disabled"],
+input[type="button"][class="disabled"],
+input[type="submit"][disabled="disabled"]:hover,
+input[type="button"][disabled="disabled"]:hover,
+input[type="submit"][disabled="true"]:hover,
+input[type="button"][disabled="true"]:hover,
+input[type="submit"][disabled]:hover,
+input[type="button"][disabled]:hover,
+input[type="submit"][class="disabled"]:hover,
+input[type="button"][class="disabled"]:hover,
+button[disabled]:hover,
+button[class="disabled"]:hover,
+.formButtonDisabled, .formButtonDisabled:hover {
+	background: transparent linear-gradient(#EEE 0%, #EEE 100%) repeat scroll 0% 0% !important;
+	color: #BBB !important;
+	border: 1px solid #DDD !important;
+	box-shadow: none !important;
+	cursor: default;
```

Comment:
Just one thing, sorry if I didn't watch it yesterday, it was late yet on Murcia. The value for cursor should be 
`cursor: not-allowed;`